init/tmpfiles.py

```diff
@@ -46,6 +50,70 @@ def list_tmpfiles_configs():
     )
 
 
+def get_specifier_map():
+    """Create mapping from specifier to value"""
+    # $TMPDIR, $TEMP, $TMP, or /tmp
+    tmpdir = tempfile.gettempdir()
+
+    uname = os.uname()
+    # machine is "x86_64", systemd wants "x86_64"
```

Comment:
Nit: did you mean `systemd wants "x86-64"`?

Reply:
yeah